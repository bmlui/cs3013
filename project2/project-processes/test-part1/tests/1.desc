eval builtin command exit, cannot use external exit, cannot use system
