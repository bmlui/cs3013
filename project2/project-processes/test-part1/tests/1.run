grep "system(" ../wshell.c -l; PATH="/bad/" ./wshell < tests/1.in
