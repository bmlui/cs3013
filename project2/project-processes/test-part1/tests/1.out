test-part1$ exit
